code-challenges/sort/challenges-03.test.js/code challenge

```diff
@@ -0,0 +1,283 @@
+'use strict';
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 1
+
+Write a function named sortBackwards that takes in an array of numbers and returns the same array, with the numbers sorted, highest to smallest.
+------------------------------------------------------------------------------------------------ */
+
+const sortBackwards = (arr) => {
+  // Solution code here...
+  sortBackwards.toStrictEqual(nums);

```

Comment:
The only method you need to complete this question is .sort(). If you need help with this please be sure to come find me (or one of the other TA's)!!!